eng/common/scripts/SemVer.ps1

```diff
@@ -143,7 +144,14 @@ class AzureEngSemanticVersion {
 
   static [string[]] SortVersionStrings([string[]] $versionStrings)
   {
-    $versions = $versionStrings | ForEach-Object { [AzureEngSemanticVersion]::ParseVersionString($_) }
```

Comment:
We shouldn't do this update at this level but instead in the constructor we should set the convention to python if the ParseLanguage is python.